content/tutorials/history.textile

```diff
@@ -31,17 +31,67 @@ We will be using channel rules in this tutorial to ensure all channels in the @p
 
 You have now enabled history for all channels in the @persisted@ namespace i.e. any channel with a name matching the pattern @peristed:*@ will store published messages to disk.
 
-h2. Step 3 - Install Ably
+h2. 
+  default: Step 3 – Install Ably
+  android: Step 3 – Set up environment and install Ably
 
 blang[java].
+  To start using Ably in your Java application, you need to include the Ably Client library. We recommend that you include the latest client library via Gradle in your project's @gradle.build@ file.
+  
+  ```[groovy]
+  apply plugin: 'application'
+  mainClassName = 'io.ably.tutorial.Example'
+
+  repositories {
```

Comment:
Android Studio has modular project structure, you have module and project level build.gradle files. It is a practice that in Android you specify "repositories" block inside project-level build.gradle. In JRE app there is only one build.gradle file since project has only one level.